meta/README.md

```diff
@@ -0,0 +1,37 @@
+# NuGet Proposal Process
+
+## How do I create a proposal?
+
+- Fork or clone https://github.com/NuGet/Home
+- Copy `meta/template.md` into `proposed/<year>/your-proposal-name.md`
+- Fill in the template with your proposal.
+- Submit a PR to the `NuGet/Home` repo
+- (Optional) Create a new discussion to talk about it https://github.com/NuGet/Home/discussions/new
+
+## How does a proposal review work?
+
+The official place for discussion for a proposal is it's pull request or a [GitHub Discussion](https://github.com/NuGet/Home/discussions). Anyone, both NuGet contributors and non-contributors can participate in the discussion and ask questions and provide constructive feedback to the proposal. Please keep in mind that only NuGet contributors are able to act upon the proposal even if other users can comment.
+
+All discussions surrounding a proposal are covered by the [.NET Foundation code of conduct](https://dotnetfoundation.org/code-of-conduct). Please keep discussions constructive, respectful, and focused on what's best for the .NET ecosystem. If discussions get heated, the NuGet team may at it's own discretion moderate the discussion to enforce the code of conduct.
+
+## What happens to a proposal?
+
+When there is consensus among the NuGet contributors and .NET community of the direction of the proposal, it will go into one of three states.
+
+- `accepted` - The proposal will be accepted and work will be scheduled to implement it.
+- `withdrawn` - The proposal has been rejected or withdrawn by the author.
+- `implemented` - The proposal has already been accepted and implemented into the product.
```

Comment:
Could we link to the corresponding folder everywhere we say `proposed`, `accepted`, `widthdrawn`, or `implemented`? This would help folks jump to where they'd like to go

Reply:
Yeah, feel free! Otherwise we can always improve these after we get this merged.